dao-technical-overview.adoc

```diff
@@ -0,0 +1,470 @@
+
+=== Blind vote payload
+Blind vote data are published when the user makes his blind vote tx and are managed in the same way as proposal payloads (append-only data).
+
+== Governance
+Governances takes place in a periodic proposal and voting cycle. A cycle consists of 4 distinct phases.
+
+=== Phases
+Phases are defined by block height. Each phase is separated with a break to avoid issues with reorgs.
+
+Here are the phases and the initial duration values (they can be changed by voting):
+
+ - Proposal phase (compensation requests, etc): 3600 blocks, about 25 days
+ - Break: 150 blocks
+ - Blind vote phase (approve/decline proposals): 450 blocks, about 3 days
+ - Break: 10 blocks
+ - Vote reveal phase: 450 blocks, about 3 days
+ - Break: 10 blocks
+ - Result phase: 10 blocks
+
+The full cycle will last 4680 blocks which is about one month if one block takes an average of 10 minutes.
+
+==== Proposal phase
+Any BSQ stakeholder can publish a temporary proposal payload during the proposal phase, as well as remove their own proposals.
+
+==== Blind Vote phase
+A BSQ stakeholder can vote on any proposal with 3 options: accept, decline or ignore. Not voting on a proposal is the same as ignoring it.
+
+The user defines how much _stake_ they want to put into their vote. The higher the stake, the higher the vote weight compared to other voters. In addition to the stake, _merit_ is added if the user has earned BSQ in previous cycles through accepted compensation requests. The merit value of each issuance ages linearly over time: it reaches 0 after 100 000 blocks (about 2 years). Aged accumulated merit is automatically added to the stake, and the sum of both is the vote weight.
+
+When creating the blind vote tx, the user also publishes the blind vote payload. We use the same linking of tx ID and payload hash to map them together.
+
+==== Vote reveal phase
+Upon entering the vote reveal phase, each voter automatically publishes their vote reveal tx. There is no fee required for this transaction beside the miner fee. No P2P network data is published.
+
+==== Vote result phase
+In the vote result phase, all nodes calculate the vote result on all proposals and apply the result to the overall BSQ state.
+
+This process uses the hash of the blind vote list from the vote reveal tx to determine the winning majority, in case users had different P2P network payloads of blind votes. The majority is calculated by stake (not merit) of the voters. Only if at least 80% of the network has the same hash is the cycle is valid, otherwise, all proposals and requests are considered rejected.
+
+A proposal is considered accepted if the required quorum and threshold are reached. Quorum is the minimum amount of accumulated vote weight in BSQ which is required. Threshold is the relationship of accepted votes to total votes. It cannot be lower then 50.01%. Each proposal type has different quorum and threshold parameters which can be changed by voting.
```

Comment:
This part seems a bit ambiguous to me. If I am understanding this correctly, this is how I would change:

Threshold is the relationship of accepted votes to total votes. Each proposal type has different quorum and threshold parameters which can be changed by voting, but threshold can never be below 50.01%.

Is that right?

Reply:
Yes better!

Reply:
I will push the change...